sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -2274,15 +2293,28 @@ SUBROUTINE MDLFLD
 !$omp parallel do private(i,j)
                DO J=JSTA,JEND
                  DO I=ISTA,IEND
-                   DENS=PMID(I,J,LL)/(RD*T(I,J,LL)*(Q(I,J,LL)*D608+1.0))      ! DENSITY
-                   GRID1(I,J) = PMTF(I,J,LL)*DENS      ! ug/kg-->ug/m3
+                   GRID1(I,J) = AVGPMTF(I,J,LL)      !ug/m3
                  ENDDO
                ENDDO
-
+               ID(1:25) = 0
+               ITHEAT     = INT(THEAT)
+               ID(19) = IFHR
+               ID(20) = 3
+               IF (IFHR==0) THEN
+                  ID(18) = 0
+               ELSE
+                  ID(18) = IFHR-1
```

Comment:
I feel "IFHR-1" is a hardwired number and a more general approach should be used

Reply:
@HuiyaChuang-NOAA These two fields are specified for AQM model only (flag aqf_on). The AQM team claims that these PM2.5 and O3 are calculated in FV3 based model for hourly averaged only. If we use  "fhzero" from model attribute, it might mislead users when fhzero is not set 1. Please let me know if you have a suggestion. @JianpingHuang-NOAA and @KaiWang-NOAA may chime in.

Reply:
@WenMeng-NOAA thanks for reply.
Suggest that you add a comment as such:

! per communication with Kai Wang, PM2.5 and O3 are computed in model as hourly average only

Reply:
I will go ahead and approve this